pages/about.html

```diff
@@ -31,13 +31,12 @@
         
         <div class="mobile-menu">
             <ul class="mobile-items">
-                <li><a href="./pages/about.html"  class="mitem">About</a></li>
-                <li><a href="#features"  class="mitem">Features</a></li>
-                <li ><a href="#partners"  class="mitem">Partners</a></li>
-                <li><a href="./pages/about.html#email"  class="mitem">Contact Us</a></li>
-                <li><a href="#speaker-container"  class="mitem">Clients</a></li>
+                <li><a href="../pages/about.html"   class="mitem">About</a></li>
+                <li><a href="../index.html#features"  class="mitem">Features</a></li>
+                <li ><a href="../index.html#partners" class="mitem">Partners</a></li>
```

Comment:
- [ ] I think here you need to add some padding to your section and kindly decarse the width and the height to the images  speaker since it is very big here and fix the side text as I asked int he mobile version it will be fixed here 

- [ ] Please adjust the styles and width & height for the container to match the design 🙏🏻🙏🏻🙏🏻

- Your design
![image](https://user-images.githubusercontent.com/58553711/181492641-20732fc1-735b-435c-9345-cb7d540159bc.png)

- Original design
![image](https://user-images.githubusercontent.com/58553711/181493409-7500f8aa-79cf-4a82-8d51-3b610f71aa0d.png)